lib/child_process.js

```diff
@@ -288,6 +292,10 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
+
+exports.execFileAsync = promisify(exports.execFile);
```

Comment:
@eljefedelrodeodeljefe Third party libraries wont be able to use this then as `process.promisify = true` wont be guaranteed. It will also need to be a CLI flag then which adds even more complications.